doc/build-netbsd.md

```diff
@@ -0,0 +1,49 @@
+NetBSD build guide
+======================
+(updated for NetBSD 7.0)
+
+This guide describes how to build bitcoind and command-line utilities on NetBSD.
+
+This guide does not contain instructions for building the GUI.
+
+Preparation
+-------------
+
+You will need the following modules, which can be installed via pkgsrc or pkgin:
+
+```
+autoconf
+automake
+boost
+db4
+git
+gmake
+libevent
+libtool
+python27
+```
+
+Download the source code:
+```
+git clone https://github.com/bitcoin/bitcoin
+```
+
+See [dependencies.md](dependencies.md) for a complete overview.
+
+### Building Bitcoin Core
+
+**Important**: Use `gmake` (the non-GNU `make` will exit with an error).
+
+With wallet:
+```
+./autogen.sh
+./configure CPPFLAGS="-I/usr/pkg/include -DOS_NETBSD" LDFLAGS="-L/usr/pkg/lib" BOOST_CPPFLAGS="-I/usr/pkg/include" BOOST_LDFLAGS="-L/usr/pkg/lib"
```

Comment:
Are the BOOST_CPPFLAGS really needed in addition to CPPFLAGS ? If so, that'd be a bug imo.

Reply:
@theuni Unfortunately, these flags are needed, I suspect due to the way Boost is installed on NetBSD via the "pkgin" system (as I understand it, the Boost project has a few peculiarities that add complexity).  Without these flags, the Bitcoin software won't compile.

Reply:
Could you please paste the error when trying to build without BOOST_CPPFLAGS / BOOST_LDFLAGS set?

It sounds like some of the checks are clobbering the usual flags. If that's the case, I'd rather fix it than work around it.